scripts/connectivity/supplementary/estimate_fc_ibc_sync_wim.py

```diff
@@ -0,0 +1,216 @@
+"""Pipeline to calculate functional connectivity and fit classifier for
```

Comment:
wim being the first name of a person, I suggest to change the name.
Same thing for next script.

Reply:
ok yes I will change it to the `mantini`

Reply:
Mantini is also a name. I'd avoid this.

Reply:
Does `external` sound ok?

Reply:
Yes.